tests/conftest.py

```diff
@@ -87,6 +87,7 @@ def pytest_addoption(parser):
         "--realcrypto", action="store_true", help="Don't mock crypto operation to save time"
     )
     parser.addoption("--runrust", action="store_true", help="Don't skip rust tests")
+    parser.addoption("--runpy", action="store_true", help="Don't skip py tests")
```

Comment:
what are `py` tests ? this should be more explicit
(and I don't see any test using this flag)

Reply:
It is used to skip tests which can't be run with rust binding

Reply:
Timestamped tests for example

Reply:
We should instead introduce a  `is_oxidized` function that return true if we currently run with the oxidation stuff, and use this to skip tests on the fly:
- by modifying the fixtures when possible (cf. what I was saying about the storage mock fixture)
- otherwise by using `@pytest.mark.skipif(is_oxidized(), reason="Test not compatible with Oxidaziton extension")` decorator